apitest/docs/api-beta-test-guide.md

```diff
@@ -408,8 +408,118 @@ The offer will be removed from other Bisq users' offer views, and paid transacti
 
 ### Editing an Existing Offer
 
-Editing existing offers is not yet supported.  You can cancel and re-create an offer, but paid transaction fees
-for the canceled offer will be forfeited.
+Offers you create can be edited in various ways:
+
+- Disable or re-enable an offer.
+- Change an offer's price model and disable (or re-enable) it.
+- Change a market price margin based offer to a fixed price offer.
+- Change a market price margin based offer's price margin.
+- Change, set, or remove a trigger price on a market price margin based offer.
+- Change a market price margin based offer's price margin and trigger price.
+- Change a market price margin based offer's price margin and remove its trigger price.
+- Change a fixed price offer to a market price margin based offer.
+- Change a fixed price offer's fixed price.
+
+_Note: the API does not support editing an offer's payment account._
+
+The subsections below contain examples related to specific use cases.
+
+#### Enable and Disable Offer
+
+Existing offers you create can be disabled (removed from offer book) and re-enabled (re-published to offer book).
+
+To disable an offer:
+```
+./bisq-cli --password=xyz --port=9998 editoffer \
+    --offer-id=83e8b2e2-51b6-4f39-a748-3ebd29c22aea \
+    --enabled=false
```

Comment:
`--enabled=false` should be `--enable=false`

Reply:
Fixed in commit a4278a4147c78ae7e4350058457af59a0083c9de.